Change NoOps to SQLite across all project folders (#959)
fermyon/developer: content/cloud/noops-sql-db.md

## Next Steps

> * For support or feature feedback, please join our [Discord channel #no-ops-db-beta](https://discord.gg/XxkZvsya). This is the place where you can provide feedback and request assistance from the Fermyon Cloud team regarding all matters related to using SQLite Databases in Spin and Fermyon Cloud.

Reviewer: How do you feel about all the extra words here?  To me "all things SQLite Database" feels more concise than "all matters related to using SQLite Databases in Spin and Fermyon Cloud."  Is the idea that non-English speakers might find the former too idiomatic?

> * For support or feature feedback, please join our [Discord channel #no-ops-db-beta](https://discord.gg/XxkZvsya). This is the place where you can provide feedback and request assistance from the Fermyon Cloud team regarding all matters related to using SQLite Databases in Spin and Fermyon Cloud.

Author: ```suggestion
* For support or feature feedback, please join our [Discord channel #no-ops-db-beta](https://discord.gg/XxkZvsya). This is the place where you can provide feedback and request assistance from the Fermyon Cloud team regarding all things SQLite Database.
```
